pkg/cluster/adminupdate_test.go

```diff
@@ -56,6 +56,7 @@ func TestAdminUpdateSteps(t *testing.T) {
 				"[Action ensureAROOperator-fm]",
 				"[Condition aroDeploymentReady-fm, timeout 20m0s]",
 				"[Condition ensureAROOperatorRunningDesiredVersion-fm, timeout 5m0s]",
+				"[Action syncClusterObject-fm]",
```

Comment:
I think it's worth adding a test for just the `MaintenanceTaskSyncClusterObject=SyncClusterObject`.

Reply:
done